docs/videos/running_all_devs.md

```diff
@@ -0,0 +1,55 @@
+# Running the All Core Devs call
+
+## Step 1. Create draft notes file for the next call
+Note: After the first All Core Devs call, this step should be performed right after the call.
+
+Create a draft notes file titled “alldevs[date].md”, with the date of the next call in YYYYMMDD format e.g. alldevs20181008.md.
```

Comment:
This section looks like a mess in the wiki, needs proper formatting.
![wiki_core_devs_running01](https://user-images.githubusercontent.com/21331903/46684618-08860c80-cbfc-11e8-8910-4069f7e3d642.png)

Reply:
oh wow, not sure how I missed that while editing. should be fixed now.